test/prometheus_stats_test.exs

```diff
@@ -8,6 +8,19 @@ defmodule PrometheusStatsTest do
     assert(String.match?(PrometheusStats.to_string(), ~r/palava_room_closed_total/))
     assert(String.match?(PrometheusStats.to_string(), ~r/palava_joined_room_total/))
     assert(String.match?(PrometheusStats.to_string(), ~r/palava_leave_room_total/))
+
+    assert(
+      String.match?(PrometheusStats.to_string(), ~r/palava_duration_room_milliseconds_bucket/)
```

Comment:
if you add something like a get_stats() function that simply calls PrometheusStats.to_string(), then all lines get a little shorter and this and the next assert probably don't take 3 lines anymore when applying the mix formatter.